### DISCUSSION TITLE
Latest Superslicer 3dconnexion support

### DISCUSSION THREAD

**Discussion by Montezuma-69** [supermerill/SuperSlicer#1572] [General]
Hello from Germany,

SuperSlicer Version 2.3.56 seems not to be based on PrusaSlicer 2.3.3 (win64).

I see a difference in die 3dconnexion space mouse support. PrusaSlicer is accepting the settings from the 3dconnexion drivers (as example speed or inverse axis), SuperSlicer is not doing it.

Cheers...

**Comment by supermerill** [comment]
what versions?
